manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: apiserver
+  name: kube-apiserver
+  namespace: openshift-monitoring
```

Comment:
side note: once https://github.com/openshift/cluster-kube-apiserver-operator/pull/502 is merged and https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/service-monitor-apiserver.yaml removed, we can switch here to the `openshift-kube-apiserver` namespace.